src/views/v2/recent-past.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * View: Recent Past View
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/recent-past.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 5.0.2
+ *
+ * @var array    $events               The array containing the events.
+ * @var string   $rest_url             The REST URL.
+ * @var string   $rest_nonce           The REST nonce.
+ * @var int      $should_manage_url    int containing if it should manage the URL.
+ * @var bool     $disable_event_search Boolean on whether to disable the event search.
+ * @var string[] $container_classes    Classes used for the container of the view.
+ * @var array    $container_data       An additional set of container `data` attributes.
+ * @var string   $breakpoint_pointer   String we use as pointer to the current view we are setting up with breakpoints.
+ */
+
+?>
+<div
+	<?php tribe_classes( $container_classes ); ?>
+	data-js="tribe-events-view"
+	data-view-rest-nonce="<?php echo esc_attr( $rest_nonce ); ?>"
+	data-view-rest-url="<?php echo esc_url( $rest_url ); ?>"
+	data-view-manage-url="<?php echo esc_attr( $should_manage_url ); ?>"
+	<?php foreach ( $container_data as $key => $value ) : ?>
```

Comment:
:no_entry_sign: Wrong escaping function. href and src attributes should be escaped by `esc_url()`, not by `esc_attr()`.

<sub>hash: b707258a560c7836d2fae404ff3f9208</sub>